model/registry.go

```diff
@@ -36,10 +36,12 @@ type Config struct {
 }
 
 // Registry of the configuration objects
+// Modification of object pointers violates thread-safety
```

Comment:
Rewrote. It doesn't break thread-safety actually anymore (since objects are re-incarnated in controller.go), but it has no effect.